src/components/LeaguesTab/LeaguesTab.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import LeaguesTabStyle from './LeaguesTabStyle';
+import LeaguesTabItem from 'Components/LeaguesTabItem';
+
+const displayName = 'LeaguesTab';
+
+const LeaguesTab = ({ leagues }) => {
+  return (
+    <LeaguesTabStyle>
+      {leagues.entrySeq().map(([key, val]) => (
+        <LeaguesTabItem
+          key={key}
+          name={val.get('name', '')}
+          country={val.get('countryCode', '')}
+        >
+          {val.get('name', '')}
+        </LeaguesTabItem>
+      ))}
+    </LeaguesTabStyle>
+  );
+};
+
+LeaguesTab.displayName = displayName;
```

Comment:
It doesn't need.
In other case, no need to use variable, that's enough `LeaguesTab.displayName = 'LeaguesTab';`